substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/c/CAnnotationProcessorCache.java

```diff
@@ -76,9 +76,16 @@
 
         @Option(help = "Exit image generation after C Annotation Processor Cache creation.")//
         public static final HostedOptionKey<Boolean> ExitAfterCAPCache = new HostedOptionKey<>(false);
+
+        @Option(help = "Output query code for target platform without executing it")//
+        public static final HostedOptionKey<Boolean> DontRunCAPQueryCode = new HostedOptionKey<>(false);
```

Comment:
I would call this option `ExitAfterQueryCodeGeneration`

Reply:
Done